packages/extension-sdk/src/connect/extension_host_api.spec.ts

```diff
@@ -379,7 +379,10 @@ describe('extension_host_api tests', () => {
     const hostApi = createHostApi()
     hostApi.error(new ErrorEvent('fake_error', errorDetail))
     expect(sendSpy).toHaveBeenCalledWith('EXTENSION_API_REQUEST', {
-      payload: errorDetail,
+      payload: {
+        ...errorDetail,
+        error: errorDetail.error.toString(),
+      },
```

Comment:
Similar to what we need to do for the TS SDK -- provide more context around the error.

Reply:
Reports back to mama about any uncaught errors in the extension. Unfortunately FF does not like to send the actual Error object over the MessageChannel : ( . Chrome is fine with it